ui/app/styles/components/node-status-light.scss

```diff
@@ -1,4 +1,4 @@
-$size: 0.75em;
+$size: 1.3rem;
```

Comment:
Can we add the icons to the status light as well? And possibly update the striped gray fill for initializing? https://share.goabstract.com/54443b43-f285-4873-8dd6-711acc63ad8c?sha=74bc51e24f125d7b56cab17f39ca674bc6c3b793

Reply:
Yep, I plan on doing this in a follow up PR (but still before we release this without it) if that's alright.